lib/Backend/DavListener.php

```diff
@@ -556,6 +562,26 @@ private function handler(array $objectData, array $calendarData, bool $isDelete)
                 $om_prefix = $this->l10N->t("Appointment pending");
             }
 
+            // create client 
+            try{
+                $client = new Client();
+                $client->setName($to_name);
+                $client->setEmail($to_email);
+                $client->setProviderId($userId);
+                $client->setDescription(" ");
```

Comment:
I've set the description to empty because it is currently not null in the database

Reply:
Yeah, I tried to set it up so the default value was an empty string, so if you didn't provide any value to the description it would set it to an empty string, but it didn't really work out.

You should be able to set it to a completely empty string, i.e. no space `""`